Bootstrap/DiagnosticLogger.ts

```diff
@@ -18,14 +18,16 @@ export class DiagnosticLogger {
             language: "nodejs",
             operation: "Startup",
             siteName: process.env.WEBSITE_SITE_NAME,
-            ikey: process.env.APPINSIGHTS_INSTRUMENTATIONKEY,
+            ikey: "unknown",
```

Comment:
I thought that while the ikey usage has been deprecated, the documentation says it will continue to work.... At least until 2025...

Reply:
Found the code below for ENV_legacyInstrumentationKey

Reply:
Yeah we always populate this one using whatever is returned from Config, that one will read all environment variables with correct priority and log warnings if needed